Automattic/wp-calypso
# Site Assembler - Color and Fonts - Remove feature flag pattern-assembler/color-and-fonts

Related #71038

## Proposed Changes

* Remove flag `pattern-assembler/color-and-fonts`

## Testing Instructions



- Create a new site with any plan
- Continue until the Design Picker
- Scroll down and click `Start designing`
- Explore and try `Colors` and `Fonts`
- Verify that `Color and Fonts screens` work as expected

<img width="1282" alt="Screenshot 2566-05-22 at 14 44 35" src="https://github.com/Automattic/wp-calypso/assets/1881481/8680b8ca-66a9-4d66-a6d9-058e020c9ff6">

<img width="1282" alt="Screenshot 2566-05-22 at 14 44 57" src="https://github.com/Automattic/wp-calypso/assets/1881481/9d279bd3-265a-4c7f-90d7-d8b4c791478f">

## Pre-merge Checklist
